Documentation/Settings.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+
+project     = TYPO3 Coding Standards Package
+release     = 0.5.x
+version     = 0.5
+copyright   = MIT license
+Description = Continuous Integration CI Core Synchronization
```

Comment:
What does that mean? I think I don't understand the "Description" really?

Reply:
Hhm, it was the best I could guess from your README. Feel free to fill in something correct. NOTE: At the moment this field isn't actually used by the docs toolchain. It is a standard bibliographic field hower, would be printed on the cover of a book, and a Sphinx extension is on the way, that will let you insert the field as variable directly into the text of your documentation. Would be nice to have such a value. But it's not mandatory. Could be left out. Your turn!